eds4jinja2/adapters/tabular_utils.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python3
+
+# tabular_utils.py
+# Date:  17/08/2020
+# Author: Eugeniu Costetchi
+# Email: costezki.eugen@gmail.com 
+
+""" a set of helper functions to easily polish the tabular data (i.e Pandas DataFrame) """
+import re
+from typing import List, Dict
+
+import pandas as pd
+import numpy as np
+import logging
+
+from pandas.core.dtypes.common import is_numeric_dtype
+
+logger = logging.getLogger(__name__)
+
+
+def invert_dict(mapping_dict: Dict, reduce_values: bool = True):
+    """
+        Invert the dictionary by swapping keys and values. In case the values are unique then the inverted dict will be
+        of the same size as the initial one. Otherwise it will be shrunk to the unique values and the keys will be
+        cumulated in a list.
+
+        The list can be reduced to single item by setting reduce_values=True.
+
+        >>> d = {"a":1, "b":2, c:1}
+        >>> reduced_d = invert_dict(d)
```

Comment:
the lost values are picked randomly, could this become a problem?

Reply:
they are not random, they are sorted; but still, the fact that values are lost is the reason for the boolean flag

Reply:
I meant random as in how python is mapping `dicts` and `sets`